crates/codegen/runtime/cargo/wasm/src/runtime/wrappers/cst/mod.rs

```diff
@@ -182,7 +182,7 @@ impl IntoFFI<ffi::Edge> for rust::Edge {
     #[inline]
     fn _into_ffi(self) -> ffi::Edge {
         ffi::Edge {
-            label: self.label.map(IntoFFI::_into_ffi),
+            label: IntoFFI::_into_ffi(self.label),
```

Comment:
nit suggestion, to match the rest of the file:

```suggestion
            label: self.label._into_ffi(),
```

Reply:
also below..

Reply:
👍